bitsofgood/pencils-for-success
# Varsh emily/159/add map markers

### Description

Added map marker icons to recipient locations

**Related Issues/PRs:** 

List of related issues/PRs and the type of association. For example:
- Closes #159 

### Test Plan

List of steps to test your proposed changes. Add screenshots if appropriate. For example: 

- Open website
- Click "Map View" button
- Map should display markers for each recipient based on the selected chapter
- When clicking on the marker, the icon should change to a pin shape, indicating it has been selected
